src/writing-your-own-application/bootloader-and-partition-table.md

```diff
@@ -1 +1,18 @@
 # Bootloader and Partition Table
+
+## Bootloader
+
+The `espflash` includes pre-built `esp-idf` [bootloaders] that are compiled using default settings, making it easy to get started without additional configuration. However, if you require more advanced functionality or custom settings, you will need to build the bootloader yourself to tailor it to your specific needs. To do that, you need to:
+1. Download and set up the [esp-idf]
+2. Make desired changes using `idf.py menuconfig`
+3. Re-build it
+4. Use the built bootloader binary with `espflash flash --bootloader <FILE>` command 
```

Comment:
Perhaps mention where we can find the bootloader file, and that we need the `.bin` to pass to espflash